packages/loudness/src/config.ts

```diff
@@ -1,16 +1,76 @@
 import dotenv from "dotenv";
-dotenv.config();
+// @ts-ignore not defined in project yet
+import getSharedConfig from 'shared-config';
+
+export interface ShareInfo {
+  name: string;
+  localizedName: string;
+  mount: string;
+  uncRoot: string;
+  cached: boolean;
+  systemRoot: string
+  matches: RegExp[];
+};
+
+export type ShareRecord = Record<string, ShareInfo>;
+
+export interface SharedConfig {
+  shares: ShareRecord;
+  environment: string;
+  printVersion: () => void;
+}
+export interface LoudnessConfig {
+  environment: string;
+  route: string;
+  shares: ShareInfo[];
+  version: string;
+  port: number;
+};
```

Comment:
I added the config types used by loudness in here, eventually this should come from the shared-config